attribution_scopes.md

```diff
@@ -89,9 +89,9 @@ If there are multiple sources whose `attribution_scopes` contains at least one o
 
 If the trigger registration's `attribution_scopes` is empty, then all sources are considered for attribution.
 
-Once an `attribution_scope_limit` is set, the last K values (where K = `attribution_scope_limit`) of `attribution_scopes` will be considered the final set of `attribution_scopes` values and any source with additional `attribution_scopes` values will be treated as if the attribution scopes were empty.
+Once an `attribution_scope_limit` is set, the last K values (where K = `attribution_scope_limit`) of `attribution_scopes` will be considered the final set of `attribution_scopes` values per reporting origin per destination site, and any source with additional `attribution_scopes` values will be treated as if the attribution scopes were empty.
 
-If a source registration is specified with a configuration that has a higher number of event states than the most recent `max_event_states` for the same reporting origin, then the source will be rejected and the registration will fail. Additionally, if the `max_event_states` field is changed in a future source registration, then all other previous pending source registrations with a different `max_event_states` will be ignored in subsequent attribution report generation flows, but will still count towards rate limits. 
+For sources that use the `attribution_scopes` feature, if the source registration is specified with a configuration that has a higher number of event states than the most recent `max_event_states` for the same reporting origin, then the source will be rejected and the registration will fail. Additionally, if the `max_event_states` field is changed in a future source registration, then all other previous pending source registrations with a different `max_event_states` will be ignored in subsequent attribution report generation flows, but will still count towards rate limits. 
```

Comment:
>...then the source will be rejected and the registration will fail.

Will an associated verbose debugging failure mode get reported?

Reply:
By "previous pending source registrations," do you mean sources that use the attribution scope feature? I guess I'm not completely clear on if we're looking at the explicit field, `max_event_states`, or the actual trigger states for the source.

Reply:
@dmdabbs yes, we do plan to support a verbose debug report that will notify of this failure.

@giladbarkan-github that statement refers to only sources that use the attribution scope feature. I will clarify in the text